dspace-server-webapp/src/test/java/org/dspace/app/rest/RelationshipRestRepositoryIT.java

```diff
@@ -2482,5 +2500,160 @@ public void putRelationshipWithJson() throws Exception {
+                                                                        .getID()))
+                                                   .andExpect(status().isCreated());
+
+        getClient(adminToken).perform(post("/api/core/relationships")
+                                                                .param("relationshipType",
+                                                                       isParentOrgUnitOf.getID().toString())
+                                                                .contentType(MediaType.parseMediaType
+                                                                    (org.springframework.data.rest.webmvc.RestMediaTypes
+                                                                         .TEXT_URI_LIST_VALUE))
+                                                                .content(
+                                                                    "https://localhost:8080/server/api/core/items/" + orgUnit2
+                                                                        .getID() + "\n" +
+                                                                        "https://localhost:8080/server/api/core/items" +
+                                                                        "/" + orgUnit3
+                                                                        .getID()))
+                                                   .andExpect(status().isCreated());
+
+        getClient().perform(get("/api/core/relationships/search/byLabel")
+                                .param("label", "isChildOrgUnitOf")
+                                .param("dso", String.valueOf(orgUnit2.getID()))
+                                .param("page", "0")
+                                .param("size", "1"))
+                    .andExpect(status().isOk())
+                    .andExpect(jsonPath("$.page", PageMatcher.pageEntryWithTotalPagesAndElements(0, 1, 1, 1)));
+
+
+    }
+
+
+    @Test
+    public void orgUnitLeftMaxCardinalityTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+        EntityType orgUnit = entityTypeService.findByEntityType(context, "OrgUnit");
+        RelationshipType isParentOrgUnitOf = relationshipTypeService
+            .findbyTypesAndTypeName(context, orgUnit, orgUnit, "isParentOrgUnitOf", "isChildOrgUnitOf");
 
+        MetadataSchema metadataSchema = metadataSchemaService.find(context, "relation");
+        MetadataFieldBuilder.createMetadataField(context, metadataSchema, "isParentOrgUnitOf", null, null).build();
+        MetadataFieldBuilder.createMetadataField(context, metadataSchema, "isChildOrgUnitOf", null, null).build();
+
+        String adminToken = getAuthToken(admin.getEmail(), password);
+
+        context.restoreAuthSystemState();
+        // Here we create our first Relationship to the Publication to give it a dc.contributor.author virtual
+        // metadata field.
+        getClient(adminToken).perform(post("/api/core/relationships")
```

Comment:
Again, `post` should cleanup after itself

Reply:
This has been updated